kitchen-tests/Gemfile.lock

```diff
@@ -59,9 +59,8 @@ GEM
     excon (0.51.0)
     faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.14)
```

Comment:
Will see if this happens again, this is why I added 2da9c1b20c11ce944835ea818a3c81566af7d739 last time.

https://travis-ci.org/chef/chef/jobs/149830214